var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -56,7 +56,7 @@ class Charliecloud(AutotoolsPackage):
     depends_on("bats@0.4.0", type="test")
 
     def autoreconf(self, spec, prefix):
-        which("sh")("autogen.sh")
+        which("bash")("autogen.sh")
```

Comment:
Is there a reason we have to redundantly tell Spack that `autogen.sh` is a Bash script? The first line is already `#!/bin/bash`.